app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -644,7 +647,7 @@ def alert_get_info(alert)
       @alert_profiles = @alert.memberof.sort_by { |p| p.description.downcase }
     end
 
-    unless @alert.expression.kind_of?(MiqExpression) # Get the EMS if it's in the expression
+    unless @alert.expression.nil? || @alert.expression.kind_of?(MiqExpression) # Get the EMS if it's in the expression
```

Comment:
This change is to add additional protection of nil value.
I agree that in this case should be changed to an if sentence.
+1